pkgs/applications/misc/rofi-rbw/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, python3Packages, fetchFromGitHub, }:
+
+python3Packages.buildPythonApplication rec {
+  pname = "rofi-rbw";
+  version = "0.5.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "fdw";
+    repo = "rofi-rbw";
+    rev = "0.5.0";
+    sha256 = "sha256-EU0rpBEiUB2/jlowq7Dr6r8iY002vWxNGoKlTP+ZyzQ=";
+  };
+
+  propagatedBuildInputs = [ python3Packages.configargparse ];
+
```

Comment:
please add pythonImportCheck and checkInputs with pytestCheckHook

Reply:
```nix
  checkInputs = [ pytestCheckHook ];
``` 
just fails? do you have a clue?

Reply:
I just had a look at the source of rofi-rbw. They do not implement any tests. 
Maybe just use
```
pythonImportsCheck = [ "rofi_rbw" ];
```

Reply:
Thanks. Can you please tell me what is the difference between `pythonImportsCheck` and `pythonImportChecks`?

Reply:
I haven't heard of `pythonImportChecks`.  Did you mean `pytestCheckHook` ? 
`pythonImportsCheck` checks, whether python can import the module from the environment.
`pytestCheckHook` runs the provided tests of the package (and fails, if no tests are provided)

Reply:
https://github.com/NixOS/nixpkgs/blob/94c97dfa94876f455c3eacd48ae1a9d0b24d5daa/pkgs/development/python-modules/nutils/default.nix#L12

Reply:
I just found it. Seems to be a typo. I just checked: It doesn't check the import.